spec/services/scoreboard_service_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe ScoreboardService do
+  describe '.update' do
+    %i[circle_check_score maneuver_participant
+       onboard_judging participant quiz_score].each do |model|
+      let!(model) { create model }
```

Comment:
Couldn't you just say `create model` here instead of using a let block? You're not using the block reference again or whatever you'd call it.

Reply:
I'm confused. Each of them _is_ used below.

That is, this loop unpacks to:

```ruby
let!(:circle_check_score) { create :circle_check_score }
let!(:maneuver_participant) { create :maneuver_participant }
#etc.
```

Reply:
Only really saves two lines, though. Maybe not worth the confusion?

Reply:
OOOoooh I get it now.  No, it's clever, leave it.